frontend/src/metabase/components/ResponsiveContainer/ResponsiveContainer.tsx

```diff
@@ -1,12 +1,9 @@
 import styled from "@emotion/styled";
 
 import type { RefProp } from "metabase/browse/types";
+import { doNotForwardProps } from "metabase/common/utils/doNotForwardProps";
```

Comment:
Now that this `doNotForwardProps` utility has been backported, we can import it here instead of redefining the function in this file